scripts/google_examples.py

```diff
@@ -11,6 +10,8 @@
 
 import yaml
 
+import scripts.utils as utils
```

Comment:
If we express this as 
```python
import utils
```
we don't need to change the way we run the scripts. We also don't need to turn `scripts` into a package, so there's no need for the `__init__.py` file.